.github/workflows/release.yml

```diff
@@ -19,45 +19,22 @@ jobs:
     - name: Install Go
       uses: actions/setup-go@v2
       with:
-        go-version: ${{ matrix.go-version }}
-        version: 1.15.8
+        go-version: 1.15.8
         stable: false
+    - name: Setup qemu
+      uses: docker/setup-qemu-action@v1
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v1
+    - name: Docker Login
```

Comment:
I've just spotted an issue whilst testing this locally. The docker images that get created are:

```
      • building docker image     image=docker.io/cue-lang/cue:v0.99.99-next-amd64
      • building docker image     image=docker.io/cue-lang/cue:v0.99.99-next-arm64
```

(I was testing against the fictional `v0.99.99`).

However, our Docker hub repository is `cuelang/cue` - i.e. it did _not_ get migrated.

Is there some way we can configure this?

Reply:
yup, let me fix it :)

Reply:
fixed!